modules/social_features/social_core/social_core.module

```diff
@@ -80,6 +81,38 @@ function social_core_modules_installed($modules) {
   }
 }
 
+/**
+ * Implements hook_entity_form_display_alter().
+ */
+function social_core_entity_form_display_alter(EntityFormDisplayInterface $form_display, array $context) {
+  $field_groups = $form_display->getThirdPartySettings('field_group');
+
+  if (!isset($field_groups['group_settings'])) {
+    return;
+  }
+
+  $fields = [];
+
+  // List of all fields which are split by section.
+  foreach ($field_groups as $field_group) {
+    $fields = array_merge($fields, $field_group['children']);
+  }
+
+  // List of all fields which aren't split by section.
+  $fields = array_diff(array_keys($form_display->getComponents()), $fields);
+
+  if ($fields) {
+    $field_groups['group_settings']['children'] = array_merge(
+      $field_groups['group_settings']['children'],
+      $fields
+    );
+
+    $form_display
+      ->setThirdPartySetting('field_group', 'group_settings', $field_groups['group_settings'])
+      ->save();
```

Comment:
We shouldn't do a save in an alter function.

Reply:
You're right. We'll need to improve it.